releases/cosmwasm/2025-02-XRPL-EVM-GMP-v6.0.4.md

```diff
@@ -290,12 +291,12 @@ node cosmwasm/submit-proposal.js execute \
   --msg "{
     \"create_pool\": {
       \"params\": {
-        \"epoch_duration\": [epoch duration],
+        \"epoch_duration\": \"[epoch duration]\",
```

Comment:
```suggestion
        \"epoch_duration\": \"$EPOCH_DURATION\",
```
give it an env var name perhaps? otherwise the escaping isn't really needed if user is substituting manually